lib/views/commerce_slack_view.ex

```diff
@@ -4,7 +4,7 @@ defmodule Aprb.Views.CommerceSlackView do
   alias Aprb.Views.{CommerceTransactionSlackView, CommerceOfferSlackView, CommerceOrderSlackView, CommerceErrorSlackView}
   def render(event, routing_key) do
     cond do
-      routing_key =~ "transaction." -> CommerceTransactionSlackView.render(event, routing_key)
+      routing_key == "transaction.failure" -> CommerceTransactionSlackView.render(event, routing_key)
```

Comment:
Thanks for adding this. I got confused when I tried to add a test case because I couldn't believe that "transaction." would be the key, and this makes a lot more sense.